### PR TITLE
Fixed _savegpr etc.

### PR DESCRIPTION
**Pull request by Julgodis**
Renamed all func_XXX to _savegpr, _restgpr, _savefpr, and _restfpr.

### CODE REVIEW

File: include/functions.h [hunk 1]
```diff
@@ -20,11 +20,11 @@ void PSMTXScale(void);
 void screenSet__12dFile_info_cFv(void);
 void dFile_info_c_NS_screenSet(void);
 void mDoExt_removeMesgFont(void);
-void func_80362210(void);
```
Comment (barzamin) [inline include/functions.h:23]: ideally we should deduplicate and group `_savegpr` and `_restgpr` functions in this file; perhaps consider a `clib.h` included into `global.h`

**Review by barzamin — APPROVED** [review]

File: include/functions.h (hunk 1 above)
Comment (notyourav) [reply include/functions.h:23]: @barzamin They are compiler intrinsics, I think it would make more sense to keep them with MWCC related stuff.

**Review by notyourav — APPROVED** [review]

File: include/functions.h (hunk 1 above)
Comment (Julgodis) [reply include/functions.h:23]: @barzamin @notyourav Moved _savegpr and _restgpr functions to a new file "mwcc.h"

**Comment by Pheenoh** [comment]
Renamed some (but not all) of these in my latest push as well. Feel free to chuck those changes in favor of the mwcc.h when you fix conflicts

**Review by Pheenoh — APPROVED** [review]

**Pull request merged**